js/src/personal/jionghao/components/Experience.tsx

```diff
@@ -0,0 +1,112 @@
+import { Title, Text, Tabs, List } from '@mantine/core'
+import styles from './Experience.module.css'
+
+export function Experience() {
```

Comment:
nit: Might want to break this up a bit more later. Would make sense to make this a folder as well.